lib/java/jna-wrapper/src/main/java/com/sun/jna/Native.java

```diff
@@ -0,0 +1,2507 @@
+/* Copyright (c) 2007-2015 Timothy Wall, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna;
```

Comment:
What version of the class file is being used here? Could you add link to the specific commit so that we know what to do on upgrades?

Reply:
As 6bb1f4f612359f1b32324ceb7674fc5c5b9989d9 puts it:

> Copy of JNA's Native as of version 5.14.0 with commented out java.awt.Window & co.

Reply:
Left a note in 5f2c2a79dd